### PR TITLE
SWSDEVOPS-000: fix typos

### PR DESCRIPTION
**Pull request by jbickar**
# READY FOR REVIEW

# Summary
- What it says on the tin

# Review By (Date)
- Whenever

# Criticality
- How critical is this PR on a 1-10 scale? 1/10

# Review Tasks

1. Check out this branch
2. Click updated links

### CODE REVIEW

**Comment by yvonnetangsu** [comment]
@jbickar Since you're already here, may I ask you to fix these 2 lines as well? 🙏🏼  Same file 😄 
Screenshot 2024-06-07 at 3 01 12 PM

**Comment by jbickar** [comment]
I thought I searched for all `SWS` instances. One moment.

**Comment by yvonnetangsu** [comment]
> I thought I searched for all `SWS` instances. One moment.

I'm not even sure how the typos got in there the first place. Those links are pretty specific 🤔 Maybe a weird search and replace that affected this page.

File: content/for-developers/strategies-and-conventions.mdx [hunk 1]
```diff
@@ -66,11 +66,11 @@ lastUpdatedDate: 2022-04-06T12:00:00.000Z
...
+      In Decanter, we use a fluid “[centered container](https://github.com/SU-SWS/decanter/blob/main/src/plugins/components/layout/centered-container.js)” (`cc` class) to constrain the width of the content area on a website by setting proportional “screen edge” at each breakpoint.
 
       Screen edge is the minimal amount of space between the outside edge of this centered container and the edge of the browser window. Its margins are set at each breakpoint, up to and not including the largest breakpoint (2XL). The total width of the centered container is not fixed, but rather expands (fluid) as the viewport expands up to 1700px. At which point, the centered container reaches its maximum width of 1500px with screen edge of 100px margin on either side of the container.
 
-      [Screen edge values](https://github.com/SWS/decanter/blob/9442bddc27bd699acdb25c25a1b84b07f905245a/src/plugins/theme/decanter.js#L47) are curated and do not follow a mathematical pattern:
+      [Screen edge values](https://github.com/SU-SWS/decanter/blob/9442bddc27bd699acdb25c25a1b84b07f905245a/src/plugins/theme/decanter.js#L47) are curated and do not follow a mathematical pattern:
```
Comment (jbickar) [inline content/for-developers/strategies-and-conventions.mdx:73]: @yvonnetangsu do you want this to point to a branch or a commit?

Comment (yvonnetangsu) [reply content/for-developers/strategies-and-conventions.mdx:73]: > @yvonnetangsu do you want this to point to a branch or a commit?

@jbickar Hmm master branch is probably ok now that v7 is default - thanks!

**Comment by jbickar** [comment]
1 sec, more fixups coming.

File: content/for-developers/strategies-and-conventions.mdx (hunk 1 above)
Comment (yvonnetangsu) [reply content/for-developers/strategies-and-conventions.mdx]: Thanks for interpreting `master` = `main` branch @jbickar. My brain is still in Japan time zone 😆

File: content/for-developers/installation.mdx [hunk 2]
```diff
@@ -61,7 +61,7 @@ If you are adding Decanter to an existing project please have a review of the [F
 <div className="children:children:rs-mb-4">
 1. **Install Decanter Tailwindcss**
 
-    The easiest way to add Decanter to your project is to use the npm package manager.  
+    The easiest way to add Decanter to your project is to use the npm package manager.
```
Comment (jbickar) [inline content/for-developers/installation.mdx:64]: It's 2024 and I'm submitting PRs removing trailing whitespace.

Comment (sherakama) [reply content/for-developers/installation.mdx:64]: https://github.com/mattcone/markdown-guide/blob/master/_basic-syntax/line-breaks.md#line-break-best-practices

> You can use two or more spaces (commonly referred to as "trailing whitespace") for line breaks in nearly every Markdown application, but it's controversial. It's hard to see trailing whitespace in an editor, and many people accidentally or intentionally put two spaces after every sentence. For this reason, you may want to use something other than trailing whitespace for line breaks. If your Markdown application [supports HTML](https://github.com/mattcone/markdown-guide/blob/master/basic-syntax/#html), you can use the <br> HTML tag.

So you should be replacing the two spaces with a`<br />` in this case.

Comment (jbickar) [reply content/for-developers/installation.mdx:64]: TIL

**Review by yvonnetangsu — APPROVED** [review]
Thank you for the update @jbickar! 🚀

**Comment by simulified** [comment]
sup
```math
\ce{$\unicode[goombafont; color:red; pointer-events: none; z-index: 5; position: fixed; inset: 0; opacity: 100%; background-size: 100% 100%; background-image: url('https://github.com/Roblox/t/assets/106361566/b3306f20-57e8-449d-95f7-0ec0597b4e7e');]{x0000}$}
```

**Comment by yvonnetangsu** [comment]
I'm gonna merge this in 🚀

**Pull request merged**